terraform/modules/iam_baseline/main.tf

```diff
@@ -8,9 +8,12 @@ resource "aws_iam_user" "cicd_member_user" {
 resource "aws_iam_group" "cicd_member_group" {
   name = "cicd-member-group"
 }
-
+#Adding checkov skips into bellow but this will probably need to be looked at in a later story#
```

Comment:
have you already created a follow on story to look at it then?
I am worried we will skip them checks and then never look at it; so I would say, if you don’t think it’s resolved, don’t skip them

Reply:
removed line 11  as we can't restrict access to resources